### ISSUE TITLE
space center grass color doesn't change in 1.8.1

### ISSUE THREAD

**Issue by Sigma88** [Kopernicus/Kopernicus#398]
I'll check that and try to fix it myself

(reported by @GregroxMun)

**Comment by GER-Space** [comment]
The buildings got a new grass shader and textures. 
This requires some new color management and a new target variable for the color.

`_GrassColor ` instead of `_Color`

The way the color is applied changed from `lerp(tex, color.rgb, color.a)` to `tex.rgb*col.rgb`

Also it got a bunch of new map names for the textures.
Ping me if you need the names, I'm only on mobile and it's painful to type for now.

**Comment by Sigma88** [comment]
yes I already have a fix ready, but thanks for the heads up :)

for references these are the options I am considering introducing into kopernicus

```
SpaceCenter
{
    groundColor = // Color (For backward compatibility, will change the "_GrassColor")
    groundTexture = // Texture (For backward compatibility, will change the "_NearGrassTexture")
    GrassMaterial
    {
        nearGrassTexture = // Texture
        nearGrassTextureOffset = // Vector2
        nearGrassTextureScale = // Vector2
        nearGrassTiling = // Float
        farGrassTexture = // Texture
        farGrassTextureOffset = // Vector2
        farGrassTextureScale = // Vector2
        farGrassTiling = // Float
        farGrassBlendDistance = // Float
        grassColor = // Color
        tarmacTexture = // Texture
        tarmacTextureOffset = // Vector2
        tarmacTextureScale = // Vector2
        tarmacColor = // Color
        opacity = // Float
        rimColor = // Float
        rimFalloff = // Float
        underwaterFogFactor = // Float
    }
}```

there are still some kinks that we need to figure out because it seems like the tarmac texture needs to have different scales depending on which part of the ksc is colored, so I need a neat way to figure out how to account for that

**Comment by GER-Space** [comment]
this list looks complete. 

1. You could ignore tarmacTextureOffset and Tiling. No reason to mess with that. 
2. I made a list of the materials of the KSC inclusing the tarmac Tiling: 
https://github.com/GER-Space/Kerbal-Konstructs/blob/master/Docs/KSC%20Materials.xlsx
You could use a dictionary for the material-name to tiling or only expose this parameter as a multiplication factor of the stock tiling. (if you want this to be edible)
3. tarmacColor is available, but has no effect. If you want I can supply a replacement shader which implement the tarmacColor parameter, because I watented/needed that for Kerbal-Konstructs. 
Now the shader ended up as a 4 material splat shader, but I can easily create a 1:1 KSC replacement shader out of it. That would be only 10 minutes of work.

**Comment by Sigma88** [comment]
Thanks for the offer but I'd rather keep things to a minimum, unless @StollD  thinks this would be a good addition

If the color does nothing, I will just remove it. And yes, I think I will make the "scale" just a multiplication of the stock value. That  seems the least intrusive solution, but I'm still toying with it.

**Comment by Sigma88** [comment]
fixed by #400
